File: testapps/sensitive-bank/src/main/java/com/example/sensitivebank/MainActivity.java
package com.example.sensitivebank;

import android.app.Activity;
import android.os.Bundle;
import android.view.WindowManager;
import android.widget.TextView;

public class MainActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getWindow().setFlags(
            WindowManager.LayoutParams.FLAG_SECURE,
            WindowManager.LayoutParams.FLAG_SECURE
        );
        TextView view = new TextView(this);
        view.setText("Fixture banking app with FLAG_SECURE");
        view.setTextSize(20f);
        setContentView(view);
    }
}

File: app/src/main/java/cz/davidstrnadel/aura/MainActivity.kt
package cz.davidstrnadel.aura

import android.os.Bundle
import android.view.WindowManager
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import androidx.core.splashscreen.SplashScreen.Companion.installSplashScreen
import cz.davidstrnadel.aura.ui.AuraAppScreen
import cz.davidstrnadel.aura.ui.theme.AuraTheme
import dagger.hilt.android.AndroidEntryPoint

@AndroidEntryPoint
class MainActivity : ComponentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        window.setFlags(
            WindowManager.LayoutParams.FLAG_SECURE,
            WindowManager.LayoutParams.FLAG_SECURE
        )
        installSplashScreen()
        super.onCreate(savedInstanceState)
        setContent {
            AuraTheme {
                AuraAppScreen()
            }
        }
    }
}
